Kernel/System/DynamicField/Driver/Set.pm

```diff
@@ -419,8 +420,8 @@ sub EditFieldValueGet {
             $DynamicField->{$Name} = { $DynamicField->{$Name}->%* };
         }
 
-        for my $Name ( sort keys $DynamicField->%* ) {
-            my $DynamicFieldConfig = $DynamicField->{$Name};
+        for my $Pair ( pairs $DynamicField->%* ) {
```

Comment:
An alternative name for `$Pair` could be `$KeyValue`, which would make the semantics more clear.